### PR TITLE
Cars ml-business vertical

### PR DESCRIPTION
**Pull request by benjaminaubin**

### CODE REVIEW

File: web/cars/cars/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+type Cars = {
+  base64: string
+}
+
+const IS_PROD = !process.env.NODE_ENV || process.env.NODE_ENV === 'development'
+const MOCK_API_CALL = IS_PROD && false
+
+const CARS_ENDPOINT :any = process.env.NEXT_PUBLIC_CARS_ENDPOINT
```
Comment (jblanche) [inline web/cars/cars/index.ts:8]: Updated that to be compatible with Vercel auto deploy logic

File: web/cars/components/Toggle.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { Switch } from '@headlessui/react'
```
Comment (jblanche) [inline web/cars/components/Toggle.tsx:1]: Do we need this file at all?

File: web/cars/next.config.js.encrypted [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (jblanche) [inline web/cars/next.config.js.encrypted:1]: Should we update this file with the new env var name ?

**Review by jblanche — CHANGES_REQUESTED** [review]

File: web/cars/next.config.js.encrypted (hunk 3 above)
Comment (benjaminaubin) [reply web/cars/next.config.js.encrypted:1]: done

**Review by jblanche — APPROVED** [review]

**Pull request merged**